RevenueCatUI/Templates/TemplateViewType.swift

```diff
@@ -14,7 +14,8 @@ private extension PaywallTemplate {
 
     var packageSetting: TemplateViewConfiguration.PackageSetting {
         switch self {
-        case .example1: return .single
```

Comment:
This and the `createView` below is all it takes to create a new template :) specify how many packages it supports, and instantiate it with the generic `TemplateViewConfiguration` /cc @aboedo.